manifests/testing/disks-images-provider.yaml.in

```diff
@@ -114,6 +114,26 @@ spec:
               - /ready
             initialDelaySeconds: 10
             periodSeconds: 5
+        - name: loopdev
+          command:
+          - sh
+          - -c
+          - |
+            while true; do
+              for i in $(seq 0 5); do
+                  rm -f /dev/loop$i
```

Comment:
Is it safe to remove the device? maybe a better approach would be to say that we continue to loop until we successfully created N loop devices?

Reply:
If I start assuming that there are other producers and consumers of loop devices in the system than this whole thing is going to need to get a lot more complicated than just the small algorithmic change you suggest.  For this constrained CI environment I would prefer to assert ownership of the host loop devices from one component and the disk-images-provider pod seems a reasonable place for that.

Reply:
What I don't understand, why can't we add that to the kind provider in kubevirtci?

Reply:
@rmohr I think it should be added as part of the tests since that is where the dependency is.  Otherwise we end up having these device created differently across different providers.